buf.gen.yaml

```diff
@@ -30,4 +30,7 @@ plugins:
   # Cpp
   - remote: buf.build/protocolbuffers/cpp:v27.1
     out: cpp/proto
-  # Java code well be generated by maven plugin
\ No newline at end of file
+  # Go
+  - remote: buf.build/protocolbuffers/go
```

Comment:
Should we specify the version? I already faced incompatibility between code generated by different protobuf versions

Reply:
Sure. However, I am not very sure about the desired version about this plugin. 
As the buf docs, it provides `buf.build/protocolbuffers/go:v1.31.0`
Do you think this version is OK? (No need to be the same with the version number for cpp/python, i.e., 27.1, right?)

Reply:
It should be 27.1 in my understanding. Just because it is the actual version of `protoc` compiler, that is used under the hood. We are [using](https://github.com/apache/incubator-graphar/blob/main/maven-projects/proto/pom.xml#L46) the same in Java.